README.md

```diff
@@ -368,6 +370,45 @@ To deploy in already existing Resource Group and Virtual Network you just need t
 resource_group_name  = "existing_rg"
 virtual_network_name = "existing_vnet"
 ```
+**Deploying GraphDB with External Application Gateway and Custom Context Path**
+
+You can deploy GraphDB without creating a new Application Gateway, allowing you to use your existing one. Additionally, you can configure a custom context path for your application. To do this, follow these steps:
+
+**_Prerequisites_**:
+- *Resource Group*: A resource group should already be created.
+- *Virtual Network*: A Virtual Network (VNet) should be set up and ready.
+- *Application Gateway*: Ensure your Application Gateway is deployed and fully operational.
+
+_Example Configuration:_
+```hcl
+context_path                  = "/graphdb"
+disable_agw                   = true
+virtual_network_name          = "your-VNet"
+resource_group_name           = "your-resource-group"
+graphdb_external_address_fqdn = "your-fqdn-or-ip"
+```
+
+_Notes_:
+- Setting `disable_agw` to true allows you to use your existing Application Gateway.
+- When using `disable_agw` you need to set `graphdb_external_adress_fqdn` as well.
```

Comment:
When using `disable_agw` .... > You need to provide `graphdb_external_adress_fqdn` when `disable_agw` is set to `true`